storage/dataflux/sequential.go

```diff
@@ -26,47 +25,51 @@ import (
 
 const (
 	// defaultPageSize specifies the number of object results to include on a single page.
-	defaultPageSize = 5000
+	defaultPageSize = 2000
 )
 
 // sequentialListing performs a sequential listing on the given bucket.
 // It returns a list of objects and the next token to use to continue listing.
 // If the next token is empty, then listing is complete.
-func sequentialListing(ctx context.Context, opts Lister) ([]*storage.ObjectAttrs, string, error) {
+func (c *Lister) sequentialListing(ctx context.Context) ([]*storage.ObjectAttrs, string, error) {
 	var result []*storage.ObjectAttrs
+	var objectsListed int
+	var lastToken string
+	objectIterator := c.bucket.Objects(ctx, &c.query)
+	objectIterator.PageInfo().Token = c.pageToken
```

Comment:
Does this approach with pageToken work if BatchSize is not a multiple of the page size? I am a bit concerned that this could lead to duplication of objects between batches. Could you double check this?

If so, I think it's fine to require that batch size be a multiple of page size (and round if not)

Reply:
In a given batch, we are returning approximately (few more) objects than in batch size.
While returning objects, we are rounding it up to a multiple of  page size. 
This will not result in duplication.

Reply:
Sounds good. Let's just add a note to the docs for BatchSize to indicate that this will be rounded.

Reply:
Added to the doc and input interface so the user is updated while using the API that the number of objects will be more and rounded up.